core-services/egov-idgen/CHANGELOG.md

```
@@ -1,13 +1,21 @@
+## 2.9.1 - 2025-05-21
+- Upgraded tracer version from 2.9.0 to 2.9.1
+- added variables in application.properties required for opentelemetry
+
 ## 2.9.0 - 2024-02-29
 - Upgraded spring boot version from 2.2.13.RELEASE to 3.2.2
 - Upgraded java version from 1.8 to 17
 - Upgraded postgresql version to 42.7.1
 - Upgraded lombok version from 1.18.8 to 1.18.22
 - Upgraded org.flywaydb:flyway-core version from 6.4.3 to 9.22.3
 
+## 2.9.1 - 2025-05-21
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)